packages/contracts/src/dollar/StakingFormulas.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.3;
+pragma solidity 0.8.16;
 
 import "./StakingShare.sol";
-import "./libs/ABDKMathQuad.sol";
+import "abdk-libraries-solidity/ABDKMathQuad.sol";
```

Comment:
I thought that the updated Solidity version made ABDK library obsolete @0xcodercrane please advise.